src/app/components/ContactGroupDropdown.js

```diff
@@ -140,7 +140,7 @@ const ContactGroupDropdown = ({ children, className, contactEmails, disabled })
 
     useEffect(() => {
         setModel(getModel(contactGroups, contactEmails));
```

Comment:
In that case we could do
```suggestion
        isOpen && setModel(getModel(contactGroups, contactEmails));
```

Reply:
That saves one `setState` and is more clear indeed